.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "eamodio.gitlens",
+    "esbenp.prettier-vscode",
```

Comment:
I thought we ran prettier through eslint. Does it need its own extension? What does this add?